packages/radio/test/radio-group.test.ts

```diff
@@ -629,3 +635,29 @@ describe('Radio Group', () => {
         expect(changeSpy.calledWith(undefined)).to.be.false;
     });
 });
+
+describe('Radio Group - late children', () => {
+    it.only('accepts frame late children', async () => {
```

Comment:
Sorry, I left the `only` in my copy/paste from hunting this down locally.
```suggestion
    it('accepts frame late children', async () => {
```
Doesn't effect anything now, but it might in the future.